app/policies/supervisor_policy.rb

```diff
@@ -1,27 +1,26 @@
 class SupervisorPolicy < UserPolicy
   def index?
-    admin_or_supervisor?
+    admin_or_supervisor? && admin_belongs_to_supervisor_org?
```

Comment:
I don't think we need this extra `admin_belongs_to_supervisor_org` method here. We basically want to say that it is okay for an Admin or a Supervisor to view the list of Supervisors, but the list of supervisors displayed is handled in the controller method already.

Reply:
Yes, I thought so...I'll remove this.